common/display/displayqueue.cpp

```diff
@@ -808,6 +808,24 @@ void DisplayQueue::RestoreVideoDefaultColor(HWCColorControl color) {
   video_lock_.unlock();
 }
 
+void DisplayQueue::SetVideoSharp(float value) {
+  video_lock_.lock();
+  requested_video_effect_ = true;
+  compositor_.SetVideoSharp(value);
+  video_lock_.unlock();
+}
+
+void DisplayQueue::GetVideoSharp(float* value, float* start, float* end) {
+  compositor_.GetVideoSharp(value, start, end);
```

Comment:
We want to have lock here too ?

Reply:
I copy this from DisplayQueue::GetVideoColor, @XiaosongWei Do we need add lock both of here? If so, I will correct both of them.

Reply:
hmm If setter needs a look than this needs to. If not we should be able to get rid of it from setter.